src/krisi/evaluate/metric.py

```diff
@@ -320,20 +318,6 @@ def __safe_set(
         else:
             self.__dict__[key] = result
 
-    # def __add__(self, other):
```

Comment:
is this needed?:)

Reply:
No, that is why I deleted it! Do you need it, I can put it back.

Reply:
ohh sorry!